src/sidebar/store/modules/selection.js

```diff
@@ -661,37 +401,23 @@ export default {
   update: update,
 
   actions: {
-    changeFocusModeUser,
     clearSelection,
     selectAnnotations,
     selectTab,
     setExpanded,
-    setFilterQuery,
     setForcedVisible,
     setSortKey,
-    toggleFocusMode,
     toggleSelectedAnnotations,
   },
 
   selectors: {
     expandedMap,
-    filterQuery,
-    filterState,
-    focusModeActive,
```

Comment:
Some of these user-focus-related selectors have not been retained as they are not used externally.